accounts-management/README.md

```diff
@@ -0,0 +1,190 @@
+# Accounts Management Package
+
+The Accounts Management package provides a comprehensive solution for Ethereum account creation, storage, and management within the Status application. It handles account generation, keystore operations, and secure storage of private keys.
```

Comment:
for Status account creation? or at least should not be ethereum as it also does chat. maybe bip44?

Reply:
Updated.